src/components/organisms/about/about.html

```diff
@@ -0,0 +1,18 @@
+<div class="container">
+  <div class="imgContent">
```

Comment:
```suggestion
  <div class="img-container">
```
クラス名はケバブケースにしてください。
こんな感じにするとより直感的なクラス名になるかと思います！
参考：https://www.wakuwakubank.com/posts/804-it-naming-convention/